src/blocks/modal.css

```diff
@@ -101,6 +101,107 @@
 
 .modal__button-close:hover {
   opacity: 1;
+  transition: opacity 0.3s ease;
+}
+
+.modal__container.modal__edit-avatar-container {
```

Comment:
`modal__edit-avatar-container` is not needed for the `avatar` form because all forms have the same styles